themes/thecyberden.omp.json

```diff
@@ -0,0 +1,128 @@
+{
+    "$schema": "https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/main/themes/schema.json",
+    "final_space": false,
+    "console_title": false,
+    "console_title_style": "folder",
+    "blocks": [
+        {
+            "type": "prompt",
+            "alignment": "left",
+            "horizontal_offset": 0,
+            "vertical_offset": 0,
+            "segments": [
+                {
+                    "type": "session",
+                    "style": "diamond",
+                    "powerline_symbol": "\uE0B0",
+                    "foreground": "#ffffff",
+                    "background": "#2e9599",
+                    "leading_diamond": "\uE0B6",
+                    "trailing_diamond": "",
+                    "properties": {
+                        "prefix": "\uFCBE ",
+                        "template": "{{ .ComputerName }}"
+                    }
+                },
+                {
+                    "type": "path",
+                    "style": "powerline",
+                    "powerline_symbol": "\uE0B0",
+                    "invert_powerline": false,
+                    "foreground": "#ffffff",
+                    "background": "#0080ff",
+                    "leading_diamond": "",
```

Comment:
as this segment is `powerline`, diamond settings should be removed

Reply:
Removed:
                    "leading_diamond": "",
                    "trailing_diamond": "\uE0B0",